docs/README.md

```diff
@@ -41,7 +106,14 @@ export default {
 
 ## Without Bundle System
 
-You can also install Kongponents into a project where there is no build system
-as long as Vue is included. Each Kongponent is packaged as a `umd.js` file, so as long as you have loaded Vue in your project the Kongponent will work as intended. You will still need to [install the base styles package](./style-guide/usage.md).
+You can also use Kongponents in a project where there is no build system as long as Vue is included on the page. Each Kongponent is packaged as a `umd.js` file, so as long as you have loaded Vue in your project the Kongponent will work as intended.
```

Comment:
do you think its worth linking to https://cli.vuejs.org/guide/build-targets.html#library ?

Reply:
👍🏼  added link